content/en/docs/zero-code/python/_index.md

```diff
@@ -45,6 +45,13 @@ Running `opentelemetry-bootstrap` without arguments lists the recommended
 instrumentation libraries to be installed. For more information, see
 [`opentelemetry-bootstrap`](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation#opentelemetry-bootstrap).
 
+{{% alert title="Note" color="info" %}} If you are using the
```

Comment:
How about this:

```suggestion
{{% alert title="Using uv?" color="warning" %}} If you are using the
```

Reply:
Sure, I'll commit the suggestion. It will look like the ["Important" at the top here](https://opentelemetry.io/docs/collector/internal-telemetry/)